stories/multi-combo-box/multi-combo-box.stories.js

```diff
@@ -0,0 +1,960 @@
+            'list',
+            'button',
+            'icon',
+            'tokenizer',
+            'dialog',
+            'bar',
+            'input'
+        ]
+    }
+};
+
+export const cozyAndCompact = () => `<div style="display:flex;height:310px">
+    <div style="display:flex;flex-direction:column;padding-right:1rem">
+        <label class="fd-form-label">
+            Multi ComboBox Cozy Mode
+        </label>
+        <div class="fd-popover">
+            <div class="fd-popover__control" aria-controls="F4GcX348a" aria-expanded="true" aria-haspopup="true">
+                <div class="fd-input-group fd-input-group--control">
+                    <div class="fd-tokenizer">
+                        <div class="fd-tokenizer__inner">
+                            <span class="fd-token" role="button">
+                                <span class="fd-token__text">
+                                    Apple
+                                </span>
+                                <button class="fd-token__close"></button>
+                            </span>
+                            <span class="fd-token" role="button">
+                                <span class="fd-token__text">
+                                    Orange
+                                </span>
+                                <button class="fd-token__close"></button>
+                            </span>
+                            <span class="fd-token" role="button">
+                                <span class="fd-token__text">
+                                    Kiwi
+                                </span>
+                                <button class="fd-token__close"></button>
+                            </span>
+                            <span class="fd-token" role="button">
+                                <span class="fd-token__text">
+                                    Banana
+                                </span>
+                                <button class="fd-token__close"></button>
+                            </span>
+                            <input class="fd-input fd-input-group__input fd-tokenizer__input" />
+                        </div>
+                    </div>
+                    <span class="fd-input-group__addon fd-input-group__addon--button">
+                        <button class="fd-input-group__button fd-button fd-button--transparent" aria-controls="F4GcX348a" aria-expanded="false" aria-haspopup="true">
+                            <i class="sap-icon--navigation-down-arrow"></i>
+                        </button>
+                    </span>
+                </div>
+            </div>
+            <div class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--dropdown fd-popover__body--dropdown-fill" aria-hidden="false" id="F4GcX348a">
+            <div class="fd-popover__wrapper">
+                <ul class="fd-list fd-list--multi-input">
+                    <li class="fd-list__item is-selected" role="option">
```

Comment:
All selected `<li>`s should have aria-selected="true"
```suggestion
                    <li class="fd-list__item is-selected" role="option" aria-selected="true">
```